spec/support/mock_successful_usps_api_call.rb

```diff
@@ -0,0 +1,8 @@
+RSpec.configure do |config|
+  config.before(:each) do |example|
+    if example.metadata[:checks_address_with_usps_api]
```

Comment:
i think it may make more sense to change the name of this to stub_usps_address_requests

The current name of "checks_address..." had me thinking that we were actually calling out to the API in the tests when in reality we're intercepting and providing mocked data.

I guess the question is whether these support assertions should be written from the point of view of the behavior of the underlying code (which checks the address) or the tests themselves (which need to stub the request). I think the way we use them currently, writing them from the POV of the test setup behavior would be more consistent.